docs/consume-packages/installing-signed-packages.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Install a signed NuGet package
+description: Describes the process of installing signed NuGet packages and configuring package signature trust settings.
+author: karann-msft
+ms.author: karann
+ms.date: 11/29/2018
+ms.topic: conceptual
+---
+
+# Install a signed package
+
+Signed packages don't require any specific action to be installed; however, if the content has been modified since it was signed, the installation is blocked with error [NU3008](../reference/errors-and-warnings/NU3008.md).
+
+> [!Warning]
+> Packages signed with untrusted certificates are considered as unsigned and are installed without any warnings or errors like any other unsigned package.
```

Comment:
we already say that in the very first line...Signed packages don't require any specific action to be installed; however, if the content has been modified since it was signed, the installation is blocked with error [NU3008]
If the signature is valid, we treat the package as unsigned from a client policy perspective.